resources/js/components/FormConditions.vue

```diff
@@ -3,22 +3,59 @@ import '/public/vendor/statamic/frontend/js/helpers.js'
 Vue.prototype.Statamic = window.Statamic
 
 export default {
-    props: ['initialData'],
+    props: {
+        initialData: {
+            type: Object,
+            required: true,
+        },
+        translations: {
+            type: Object,
+            default: () => ({}),
+        },
+        callback: {
+            type: Function,
+        },
+    },
     data() {
         return {
             formData: this.initialData,
+            success: false,
+            error: false,
```

Comment:
The developer can use these to show/hide validation on the frontend apart from the notify.